chpl/chpl-service/src/main/java/gov/healthit/chpl/activity/history/explorer/CertificationResultContainsSvapActivityExplorer.java

```diff
@@ -41,7 +43,7 @@ public CertificationResultContainsSvapActivityExplorer(ActivityDAO activityDao,
 
     @Override
     @Transactional
-    public ActivityDTO getActivity(ListingActivityQuery query) {
+    public List<ActivityDTO> getActivities(ListingActivityQuery query) {
```

Comment:
Does nothing call the original version of this method any more?

Reply:
No... but I am going to think about if there is a better way to make this code change. This is the first activity explorer that has needed to return more than one activity.